app/src/test/java/org/simple/clinic/textInputdatepicker/TextInputDatePickerUpdateTest.kt

```diff
@@ -71,4 +83,24 @@ class TextInputDatePickerUpdateTest {
             )
         )
   }
+
+  @Test
+  fun `when done button is clicked, then show date validation errors`() {
+    val model = defaultModel
+        .yearChanged("2019")
+        .dayChanged("04")
+        .monthChanged("34")
+    val minDate = LocalDate.of(2019, 4, 4)
```

Comment:
FYI, we usually use `parse` to define local dates. Let's replace `LocalDate.of` with `LocalDate.parse` in this PR
```kt
LocalDate.parse("2018-01-01")
```